chromium_src/chrome/browser/ui/views/tabs/tab_strip.cc

```diff
@@ -4,10 +4,21 @@
  * You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 #include "chrome/browser/ui/views/tabs/tab_strip.h"
```

Comment:
are all of these includes still relevant?

Reply:
Removed that one - the others all seem to be necessary.